app/views/quotes/index.html.erb

```diff
@@ -0,0 +1,11 @@
+<h2>Search results for: <%= params[:query] %></h2>
```

Comment:
I don't believe that you'll have access to params here. This is something you can pass over from your quotes#index action